sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -252,6 +252,7 @@ class StreamExecution(
    */
   private def runBatches(): Unit = {
     try {
+      sparkSession.sparkContext.setJobGroup(runId.toString, getBatchDescriptionString)
```

Comment:
note here it is `setJobGroup(..., interruptOnCancel = false)`. Should we add a sql conf to for `interruptOnCancel`?